src/ert/gui/model/snapshot.py

```diff
@@ -429,6 +431,16 @@ def _fm_step_data(
                 )
                 # There is no method for truncating microseconds, so we remove them
                 delta -= timedelta(microseconds=delta.microseconds)
+                if delta < timedelta():
+                    global warn_once  # noqa: PLW0603
+                    if warn_once:
+                        warn_once = False
+                        logger.warning(
+                            "Negative duration in snapshot encountered. "
+                            f"start_time={start_time} end_time={node.data.get(ids.END_TIME)} "
+                            f"delta={delta}"
+                        )
```

Comment:
@xjules showed me a trick in 2023, and now I am showing it to you =)
```suggestion
                        end_time = node.data.get(ids.END_TIME)
                        logger.warning(
                             "Negative duration in snapshot encountered. "
                             f"{start_time=} {end_time=} {delta=}"
                        )
```

Reply:
in this instance it is not that useful. f"{object=} gives object.\_\_repr__ while I want object.\_\_str__
\_\_str__ `'2025-01-01 00:00:00'` 
\_\_repr__ `'datetime.datetime(2025, 1, 1, 0, 0)'`

Reply:
I don't get why this shouldn't work @JHolba ?

Reply:
it works, but you get different text. I want this representation: `'2025-01-01 00:00:00'`.

you can try this in a repl
```python
from datetime import datetime
start = datetime.now()
print("print using {start=}: " f"{start=}")
print("print using start={start}: " f"start={start}")
```